app/views/orders/receipt.html.erb

```diff
@@ -1,5 +1,6 @@
 <h1>RECEIPT</h1>
 
 <%= render "display_nonpending_order_items" %>
-
+<h5>Purchase Time: <%= @order.created_at.strftime ("%c") %></h5>
+<h5>Order Status: <%= @order.status %></h5>
```

Comment:
If this is going to be in the receipt, wouldn't it also make sense to add it to the pending order items table, so the customer will see the change from pending to paid?